mvg/mvg.py

```diff
@@ -396,7 +396,7 @@ def create_measurement(
 
     # in example
     def read_measurements(self, sid: str):
```

Comment:
To be consist with similar functions in the package, `list_sources()` and `list_analyses()`, I suggest we rename this function to `list_measurements()`